jwala-services/src/main/java/com/cerner/jwala/service/jvm/impl/JvmServiceImpl.java

```diff
@@ -622,7 +638,15 @@ private void secureCopyJvmConfigJar(Jvm jvm, String jvmConfigJar, User user) thr
 
     private void deployJvmConfigJar(Jvm jvm, User user, String jvmJar) throws CommandFailureException {
         final String parentDir = ApplicationProperties.get("remote.paths.instances");
-        CommandOutput execData = jvmControlService.controlJvm(
+        CommandOutput execData = jvmControlService.executeCreateDirectoryCommand(jvm, parentDir+"/"+jvm.getJvmName());

```

Comment:
I deleted the following as part of my fix, can you double check with master? This will cause the creation of additional empty jvm  backup folder, I think you need to remove this.
jvmControlService.executeCreateDirectoryCommand(jvm, parentDir+"/"+jvm.getJvmName());